MaxText/configs/base.yml

```diff
@@ -135,6 +135,9 @@ remat_policy: 'full'
 scan_layers: True
 param_scan_axis: 1
 attention: 'autoselected' # Supported attention: autoselected, dot_product, flash, cudnn_flash_te
```

Comment:
Can we add a comment clarifying the difference between attention and attention_type - which is that:
attention refers to the computation methodology
attention_type refers to different variants of the attention math
Please also list the options for attention_type here - global, local_sliding

Reply:
Done